### DISCUSSION TITLE
https://scratch-status.toakiryu.com/ja

### DISCUSSION THREAD

**Discussion by giscus** [toakiryu/gform-quick-submit#15] [General]
# https://scratch-status.toakiryu.com/ja

Scratchサービスの稼働状況をリアルタイムで監視し、稼働時間、バージョン、サーバー負荷、データベースの健康状態などの詳細なステータス情報を提供します。Scratchの主要なサービスとキャッシュ接続の状態を確認できます。

https://scratch-status.toakiryu.com/ja

**Comment by syun-scratch** [comment]
クラウド変数の状態が確認できる機能も追加して欲しいです！

**Reply by toakiryu** [reply]
その機能は現在開発段階にあります。今後動作が安定すれば正式実装されます。このステータス情報は Scratch API ( Scratchが提供するAPI ) を使用して情報を取得しています。しかし、データの項目ごとに公式からの説明などがなく、どの値がどの機能に関係しているのかが、把握できていない状況です。現在 Scratch チームに APIのレスポンス情報について問い合わせています。

**Reply by syun-scratch** [reply]
なるほど。ご回答ありがとうございます。

**Reply by toakiryu** [reply]
申し訳ないのですが、サイトに設置されたコメントフォームのリポジトリを設定間違えてしまっていました。正しくは[こちら](https://github.com/toakiryu/scratch-status/discussions/5)のディスカッションをご利用ください

**Reply by toakiryu** [reply]
クラウド変数のサーバーがオンラインまたはオフラインの情報を表示する項目を追加しました。現在はまだ、クラウド変数サーバーの詳細な情報を取得する方法がないため詳細は今しばらくお待ちください。

![スクリーンショット 2024-12-31 23 27 26](https://github.com/user-attachments/assets/5082d8a2-922c-4ec5-a073-9ee34c340faf)
